tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -171,6 +171,15 @@ public:
 
       RValue GetNonOwningCopy() { return RValue(fField, fObjPtr, false); }
 
+      template <typename T>
+      void *Release()
```

Comment:
Shouldn't it be?
```suggestion
      T *Release()
```

Reply:
I find the `T *` the safer default. You can still call `Release<void>()`

Reply:
@jblomer I am confused both you and Vincenzo are advocating for `T*` while github says that `void*` was merged !?

Reply:
Ping on this!  Indeed it should be
```suggestion
      T *Release()
```
!